tests/test_countries.py

```diff
@@ -65,9 +65,12 @@ class TestCountries:
             ("Solomon Islands", {}, "SB"),  # en
```

Comment:
```suggestion
            ("Solomon Islands", {}, "SB"),  # en
             # However, in cases where island is singular instead of plural,
            # there can be confusion.
            ("Solomon Island Nationals", {}, None),  # en
```